aten/src/ATen/native/native_functions.yaml

```diff
@@ -4355,6 +4355,16 @@
   dispatch:
     CompositeExplicitAutograd: _unsafe_view
 
+# _unsafe_reshape_alias is used in cases where reshape can be performed as
+# a view operation and we want to preserve gradients.
+
+- func: _unsafe_reshape_alias(Tensor(a) self, int[] size, int[] strides) -> Tensor(a)
```

Comment:
Why do you call this one "_unsafe" ?
It should be an actual view op and be added to the list in `tools/autograd/gen_view_or_inplace.py`

Reply:
`_unsafe_reshape_alias` essentially does the same thing as `view` does but without any of the checks to avoid recalculating shape and computing strides. 

The original implementation I did to skip the `view` overhead lost the autograd support (it just called `alias_with_sizes_and_strides` directly from the `reshape` operator instead of dispatching). So this is a first attempt at preserving autograd while still removing the unnecessary overhead of calculating strides (see #55126 for the motivation).

Is there a doc I can look at to find out more about view ops versus regular ops? 

I don't expect this function to be called by end users or outside of the `reshape` operator since it's skipping bounds checking.